lib/commands/web.js

```diff
@@ -66,4 +66,24 @@ extensions.checkForAlert = async function () {
   return false;
 };
 
+extensions.waitForAtom = async function (promise) {
```

Comment:
@imurchie can correct me if i'm wrong, but this whole thing looks equivalent to the much simpler:

```js
// Override this for WDA by essentially doing nothing
extensions.waitForAtom = async function (promise) {
  let res = await promise;
  return this.parseExecuteResponse(res);  
};
```

(since the `.then` and `.catch` will already be on the promise, and `await`ing the promise will throw in this context if the promise is rejected.